assets/sass/_tables.scss

```diff
@@ -108,3 +134,66 @@ Style guide: Tables.2 Accessibility
     border-bottom: solid 2px $grey;
   }
 }
+
+
+.calendar-table {
+  width: 100%;
+  margin-top: 2 * $base-spacing;
+
+  caption {
+    font-size: rem(24);
+    font-weight: 700;
+    margin: 0 0 $small-spacing;
+    text-align: left;
+  }
+
+  tr {
+    border-bottom: $tiny-spacing solid $white;
+  }
+
+  th {
+    color: $aqua;
+    display: block;
+    padding: $small-spacing $base-spacing;
+    text-transform: uppercase;
+    width: 100%;
+
+    @include media($mobile) {
+      display: table-cell;
+      width: 6 * $base-spacing;
+    }
+
+    span {
+      @extend h2;
+      display: block;
+      margin: 0;
+    }
+  }
+
+  td {
+    display: block;
+    @extend h4;
```

Comment:
Only concerned about these `@extends` — might refactor those to the right `font-size` and margins to cleanup the output CSS.